pkgs/applications/audio/cozy-audiobooks/default.nix

```diff
@@ -0,0 +1,86 @@
+{ stdenv, fetchFromGitHub
+, ninja
+, boost
+, meson
+, pkgconfig
+, wrapGAppsHook
+, appstream-glib
+, desktop-file-utils
+, gtk3
+, glib
+, gst_all_1
+, gobjectIntrospection
+, python3Packages
+, file
+, cairo
+, sqlite
+, gettext
+, gnome3
+}:
+
+stdenv.mkDerivation rec {
+  name = "cozy-${version}";
+  version = "0.6.3";
+
+  src = fetchFromGitHub {
+    owner = "geigi";
+    repo = "cozy";
+    rev = version;
+    sha256 = "0xs6vzvmx0nvybpjqlrngggv2x8b2ky073slh760iirs1p0dclbc";
+  };
+
+  postPatch = ''
+    chmod +x data/meson_post_install.py
+    patchShebangs data/meson_post_install.py
+    substituteInPlace cozy/magic/magic.py --replace "ctypes.util.find_library('magic')" "'${file}/lib/libmagic${stdenv.hostPlatform.extensions.sharedLibrary}'"
+  '';
+  postInstall = ''
+      wrapProgram $out/bin/com.github.geigi.cozy \
+      --prefix PYTHONPATH : "$PYTHONPATH:$(toPythonPath $out)"
+      ln -s $out/bin/com.github.geigi.cozy $out/bin/cozy-audiobooks
```

Comment:
The author intended for the executable to be rddn'd and I think this won't match the `Exec` in the .desktop file either.

Reply:
you mean renamed to `cozy`? i can do that